inference-engine/tests/functional/plugin/shared/include/behavior/exec_graph_info.hpp

```diff
@@ -72,7 +72,7 @@ TEST_P(ExecGraphTests, CheckExecGraphInfoBeforeExecution) {
         const auto originalLayers = function->get_ops();
         std::map<std::string, int> originalLayersMap;
         for (const auto &layer : originalLayers) {
-            if (layer->description() == "Result")
+            if (layer->description() == "Result" && targetDevice != CommonTestUtils::DEVICE_CPU)
```

Comment:
@dmitry-gorokhov why this condition was added?

Reply:
@mandrono Could you please comment?

Reply:
Because of this change we have to write even more strange code like https://github.com/openvinotoolkit/openvino/pull/5366/files#diff-2ef05639eaefff6facdebd560ec3e4a69eb5e0ecaefa2ab98d5aef485a21985aR74-R80

We need aligned behavior..

Reply:
why do we have condition ```layer->description() == "Result"``` at all for this tests? looks like here supposed that we don't dump ```Result``` node when call ```GetExecGraphInfo()``` but we do this and we did it before ngraph migration.
for previous version it's work because output node doesn't contain original_names and we don't enter in ```for_each``` cycle. But after ngraph migration output node contain original_names and test failed, because can't find original_name in ```originalLayersMap```. 
Should not we remove this condition ```layer->description() == "Result"``` at all?
If I missed something, correct me please

Reply:
Could you please create pull request with this condition removal? Let's see how it works for other plugins